### PR TITLE
Review branch added

### PR DESCRIPTION
**Pull request by Lezanders**
Created the Reviews subpage and corrected the Requested spaces

### CODE REVIEW

File: pages/review.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+          <div class="reviewCard">
+            <img 
+              src="../images/ProfileImg/bruna.jpg" 
+              alt="Woman with a black dress and face witth a lot os piercings" 
+              class="profileImg"
+            >
+            <div class="reviewText">
+              <h4>Bruna:</h4> 
+              <p>
+                <br>
```
Comment (basiacarvalho) [inline pages/review.html:46]: You shouldn't use "br" for getting spaces between paragraphs. Instead you should use "margin" or "padding"

File: pages/review.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+            </div>
+          </div>
+
+
+
+        </div>
```
Comment (basiacarvalho) [inline pages/review.html:63]: If I see correctly, here is the closing "div" for all "reviewCards" and you put here just one card, the rest you places below.

File: pages/review.html [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+              <br><br>
+              <div class="stars">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+              <img src="../images/ProfileImg/star.svg" alt="" class="reviewStar">
+            </div>
+          </div>
+
```
Comment (basiacarvalho) [inline pages/review.html:60]: You can't have empty lines in html document. I have formatted the above parts as an example for you. Look at the code above and try to adjust the code below accordingly.

**Review by basiacarvalho — CHANGES_REQUESTED** [review]
I left comments for you and formatted a fragment of your code as an example.

File: css/style.css [hunk 4]
```diff
@@ -864,3 +866,18 @@ grid-template-columns: 1fr 1fr;
 .stars{
   text-align: right;
 }
+
+@media (max-width: 430px) {
```
Comment (basiacarvalho) [inline css/style.css:870]: Try to adjust to our @media breakpoints.
Here is an example from our code:

breakpoint for tablet starts here
```
@media (min-width: 768px) {
}
```
breakpoint for desktop starts here
```
@media (min-width: 1200px) {
 }
```

**Review by basiacarvalho — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 4 above)
Comment (Lezanders) [reply css/style.css:870]: I just ajusted to the mobile L and smaller because was the only one that was "deforming" the page.

**Comment by basiacarvalho** [comment]
Unfortunately, you didn't finish:
- html file format - mainly empty lines, incorrect indentations
- html structure - tags closed in the wrong places
- each section should have a heading (one of them lacked it)
I've changed the most urgent issues and will approve the pull request now.

**Review by basiacarvalho — APPROVED** [review]
Unfortunately, you didn't finish:
- html file format - mainly empty lines, incorrect indentations
- html structure - tags closed in the wrong places
- each section should have a heading (one of them lacked it)
I've changed the most urgent issues and will approve the pull request now.

**Pull request merged**